src/main/java/com/sprint/mission/discodeit/service/basic/BasicChannelService.java

```diff
@@ -0,0 +1,141 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.dto.channel.ChannelCreateDTO;
+import com.sprint.mission.discodeit.dto.channel.ChannelFindDTO;
+import com.sprint.mission.discodeit.dto.channel.ChannelUpdateDTO;
+import com.sprint.mission.discodeit.dto.readstatus.ReadStatusCreateDTO;
+import com.sprint.mission.discodeit.dto.user.UserFindDTO;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ChannelType;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.MessageRepository;
+import com.sprint.mission.discodeit.repository.ReadStatusRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.service.ChannelService;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor //final 혹은 @NotNull이 붙은 필드의 생성자를 자동 생성하는 롬복 어노테이션
+public class BasicChannelService implements ChannelService {
+    private final ChannelRepository channelRepository;
+    private final UserRepository userRepository;
+    private final ReadStatusRepository readStatusRepository ;
+    private final MessageRepository messageRepository;
+    //Service
+    private final ReadStatusService readStatusService;
+
+    @Override
+    public Channel createPrivateChannel(ChannelCreateDTO channelCreateDTO, ChannelType type) {
+        Channel channel = new Channel(channelCreateDTO, ChannelType.PRIVATE);
```

Comment:
ChannelType에 따라 메서드를 나누었다면 ChannelType 파라미터는 제거해도 될 것 같아요.  
```suggestion
    @Override
    public Channel createPrivateChannel(ChannelCreateDTO channelCreateDTO) {
        Channel channel = new Channel(channelCreateDTO, ChannelType.PRIVATE);
```